src/lua/help_en_US.lua

```diff
@@ -159,117 +159,90 @@ When you’re ready to go to the next screen, enter <tutorial("next")>.
 ]];
 
 [[
-Tutorial -- Screen #6 -- Delimiters
-===================================
-
-This is just to prepare for later exercises
-which will go over many lines. There is a
-Tarantool instruction that means <don’t execute
-every time I type Enter; wait until I type a
-special string called the "delimiter".>
-More in the Tarantool manual:
-http://tarantool.io/en/doc/2.1/reference/reference_lua/console/#console-delimiter
-
-Request #6 is:
-
-console = require("console"); console.delimiter("!")
-----------------------------------------------------
-
-It’s not an exercise -- just do it.
-Cancelling the delimiter could be done with
-console.delimiter("")!
-but you’ll see "!" in following exercises.
-
-You'll need a custom delimiter only in the online trial console at
```

Comment:
https://tarantool.io/en/try-dev/ still requires to use a custom delimiter. So why do you propose to remove this section?

Reply:
Decided to leave the delimiter for now. Filed https://github.com/tarantool/try/issues/29